### PR TITLE
fix: using resnet features and not logits

### PR DESCRIPTION
**Pull request by samsja**

### CODE REVIEW

**Review by alexcg1 — APPROVED** [review]
LGTM @samsja . I was just using the classification layer, right? And I should've gone a layer lower?

Just asking for my own understanding

**Comment by samsja** [comment]
> LGTM @samsja . I was just using the classification layer, right? And I should've gone a layer lower?
> 
> Just asking for my own understanding

Yes exactly, we should perform a search on the feature level
